Makefile

```diff
@@ -1215,6 +1220,7 @@ endif
 	@echo '                           (note: the name in the configuration file is not overwritten)'
 	@echo '  L=[PATH]:[PATH]:..     - colon-separated list of paths to external libraries'
 	@echo '  P=[PATH]:[PATH]:..     - colon-separated list of paths to external platforms'
+	@echo '  E=[PATHSUFFIX]:..      - colon-separated list of paths (or suffixes) of libraries that shall be excluded'
```

Comment:
With the latest update this message needs to be now `E=[PATH]:[PATH]` and `(or suffixes)` needs to be removed, right?

Reply:
sorry, I somehow missed that one :+1: